### PR TITLE
Add election view function with set size as parameter

### PR DESCRIPTION
**Pull request by asaj**
### Description

Allows users to pull the validator set that would be elected right now, taking the minimum and maximum validator set sizes as input. 

### Tested

Not tested.

### Other changes

None
### Related issues

- Needed for direct announce changes

### Backwards compatibility

Backwards compatible

### CODE REVIEW

File: packages/protocol/contracts/governance/Election.sol [hunk 1]
```diff
@@ -817,9 +817,22 @@ contract Election is
...
+  }
+
+  /**
+   * @notice Returns a list of elected validators with seats allocated to groups via the D'Hondt
+   *   method.
+   * @return The list of elected validators.
+   * @dev See https://en.wikipedia.org/wiki/D%27Hondt_method#Allocation for more information.
+   */
+  function electValidatorSigners(uint256 minElectableValidators, uint256 maxElectableValidators)
+    external
```
Comment (m-chrzan) [inline packages/protocol/contracts/governance/Election.sol]: Wouldn't this function need to be `public` now since it's called from within the contract?

**Review by m-chrzan — APPROVED** [review]
Just one potential issue, which should (hopefully!) be caught by governance tests.

File: packages/protocol/contracts/governance/Election.sol (hunk 1 above)
Comment (asaj) [reply packages/protocol/contracts/governance/Election.sol]: Definitely

Comment (asaj) [inline packages/protocol/contracts/governance/Election.sol]:
```suggestion
    public
```

File: packages/protocol/contracts/governance/Election.sol [hunk 2]
```diff
@@ -817,9 +817,22 @@ contract Election is
    * @notice Returns a list of elected validators with seats allocated to groups via the D'Hondt
    *   method.
    * @return The list of elected validators.
-   * @dev See https://en.wikipedia.org/wiki/D%27Hondt_method#Allocation for more information.
    */
   function electValidatorSigners() external view returns (address[] memory) {
+    return electValidatorSigners(electableValidators.min, electableValidators.max);
```
Comment (asaj) [inline packages/protocol/contracts/governance/Election.sol]:
```suggestion
    return electNValidatorSigners(electableValidators.min, electableValidators.max);
```

File: packages/protocol/contracts/governance/Election.sol [hunk 3]
```diff
@@ -817,9 +817,22 @@ contract Election is
...
+  }
+
+  /**
+   * @notice Returns a list of elected validators with seats allocated to groups via the D'Hondt
+   *   method.
+   * @return The list of elected validators.
+   * @dev See https://en.wikipedia.org/wiki/D%27Hondt_method#Allocation for more information.
+   */
+  function electValidatorSigners(uint256 minElectableValidators, uint256 maxElectableValidators)
```
Comment (asaj) [inline packages/protocol/contracts/governance/Election.sol]:
```suggestion
  function electNValidatorSigners(uint256 minElectableValidators, uint256 maxElectableValidators)
```

**Pull request merged**